### ISSUE TITLE
onLongPress，I hope to be able to pass in custom methods

### ISSUE THREAD

**Issue by JieNengi** [zezo357/flutter_meedu_videoplayer#99]
In some scenarios, onLongPress requires custom methods

**Comment by zezo357** [comment]
I am currently working on something else, if you can provide a pr I would happily review it 



But if you can't let me know so I implement it when I have time

**Comment by zezo357** [comment]
example from new release
```
  final _meeduPlayerController = MeeduPlayerController(
      controlsStyle: ControlsStyle.custom,
      enabledOverlays: const EnabledOverlays(volume: false),
      customCallbacks: CustomCallbacks(onLongPressStartedCallback: (MeeduPlayerController controller) {
      })
      
      );

```

**Comment by zezo357** [comment]
fixed on `4.2.11-dev-3` please check and reopen it there is any other problem
